scripts/dev

```diff
@@ -2,11 +2,20 @@
 
 npm_bin=$(npm bin)
 
+export POSTGRAPHQL_ENV=development
 export DEBUG=${DEBUG-postgraphql*,-postgraphql:graphql,-postgraphql:postgres:explain}
 
+# Run Nodemon and watch the PostGraphQL source code for changes.
 $npm_bin/nodemon \
   --watch src \
   --ignore __tests__ \
   --ignore __mocks__ \
+  --ignore src/postgraphql/graphiql \
   --ext js,ts \
-  --exec "$npm_bin/ts-node --ignore node_modules --disableWarnings src/postgraphql/cli.ts --schema a,b,c --show-error-stack json --watch $@"
+  --exec "$npm_bin/ts-node --ignore node_modules --disableWarnings src/postgraphql/cli.ts --schema a,b,c --show-error-stack json --watch $@" & \
```

Comment:
I believe it is because we are running in parallel with the `PORT=5001 npm --prefix src/postgraphql/graphiql run start > /dev/null` script below (note the `&`). I may be wrong though.